filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt

```diff
@@ -623,4 +545,464 @@ filmboxlive.com##.mobox-wrapper:style(overflow: auto !important;)
 ! Rule "galopuje.pl,sklepbiker.pl##.modal-open:style(padding-right: 0 !important; overflow: auto !important;)" converted to: galopuje.pl,sklepbiker.pl#$#.modal-open { padding-right: 0 !important; overflow: auto !important; }
 galopuje.pl,sklepbiker.pl##.modal-open:style(padding-right: 0 !important; overflow: auto !important;)
 ! Rule "powiatslubicki.pl##.modal-open:style(overflow: auto !important;)" converted to: powiatslubicki.pl#$#.modal-open { overflow: auto !important; }
-powiatslubicki.pl##.modal-open:style(overflow: auto !important;)
\ No newline at end of file
+powiatslubicki.pl##.modal-open:style(overflow: auto !important;)
+! 4lomza.pl#$#body.polityka-cookies { margin-top: 0 !important; } is excluded by: #$#

```

Comment:
Why exclude CSS?

Reply:
@tvinzz please change trust-level

Reply:
anyway I revert filters to `##*:style()`

Reply:
Added `high` trust level for `217_PolskiFilterCookies`